### PR TITLE
ObsPy: Add introduction and install instructions

### PR DESCRIPTION
**Pull request by core-man**
#40﻿

### CODE REVIEW

File: source/obspy/introduction.rst [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+5. 匹配地震和台站元数据
+6. 波形数据处理
+7. 波形数据分析
+
+传统的地震学软件往往只具有以上的一个或几个功能，为此我们需要学习和使用多个软件，并且编写多个脚本。
+
+ObsPy 的优势如下：
+
+1. 可以完成以上所有流程
+2. 可以使用丰富 Python 软件库
```
Comment (seisman) [inline source/obspy/introduction.rst:28]:
```suggestion
2. 可以使用丰富的 Python 软件库
```

File: source/obspy/introduction.rst [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+3. 支持目前绝大多数地震学数据格式的读写
+4. 已有许多基于 ObsPy 开发的软件包
+
+引用
+----
+
+如果使用 ObsPy (直接使用或作为其他程序包的依赖包) 开展研究工作，发表了学术文章，可以引用以下参考文献的任意一篇。
```
Comment (seisman) [inline source/obspy/introduction.rst:35]:
```suggestion
如果使用 ObsPy (直接使用或作为其它程序包的依赖包) 开展研究工作，发表了学术文章，可以引用以下参考文献的任意一篇。
```

File: source/obspy/introduction.rst [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+- Krischer, L., Megies, T., Barsch, R., Beyreuther, M., Lecocq, T., Caudron, C., & Wassermann, J. (2015).
+  ObsPy: a bridge for seismology into the scientific Python ecosystem.
+  *Computational Science & Discovery*, 8(1), 014003.
+  https://doi.org/10.1088/1749-4699/8/1/014003
+
+此外，也可引用实际使用的 ObsPy 版本：
+
+`已发布的 ObsPy 版本 <https://zenodo.org/search?ln=en&p=obspy&sort=mostrecent>`__
+(例如 ObsPy 1.2.0: `10.5281/zenodo.3674646 <http://dx.doi.org/10.5281/zenodo.3674646>`__)
```
Comment (seisman) [inline source/obspy/introduction.rst:55]: As we mentioned that we're working on v1.2.2, perhaps we only need to list the v1.2.2 DOI here.

File: source/obspy/install.rst [hunk 4]
```diff
@@ -0,0 +1,33 @@
+安装
+====
+
+我们强烈推荐使用 Anaconda 来安装 ObsPy，这也是 ObsPy 的 `官方推荐 <https://github.com/obspy/obspy/wiki#installation>`__。
+
+1.  安装 Anaconda
```
Comment (seisman) [inline source/obspy/install.rst:6]: Related to #49

File: source/obspy/introduction.rst (hunk 2 above)
Comment (core-man) [reply source/obspy/introduction.rst:35]: I think now we usually use 其他 instead of 其它.  I asked a teacher about this issue, and he told me the same thing.

Comment (seisman) [reply source/obspy/introduction.rst:35]: Good to know.

File: source/obspy/install.rst [hunk 5]
```diff
@@ -0,0 +1,32 @@
+安装
+====
+
+我们强烈推荐使用 Anaconda 安装 ObsPy，这也是 ObsPy 的 `官方推荐 <https://github.com/obspy/obspy/wiki#installation>`__。
```
Comment (seisman) [inline source/obspy/install.rst:4]:
```suggestion
我们强烈推荐使用 Anaconda 安装 ObsPy，这也是 ObsPy `官方推荐的安装方式 <https://github.com/obspy/obspy/wiki#installation>`__。
```

File: source/obspy/install.rst [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+5.  检查安装是否成功::
+
+    终端键入 ``python`` 启动 Python 解释器，若能导入 obspy 模块则表示安装成功::
+
+        $ python
+        Python 3.8.6 | packaged by conda-forge | (default, Oct  7 2020, 19:08:05) 
+        [GCC 7.5.0] on linux
+        Type "help", "copyright", "credits" or "license" for more information.
+        >>> import obspy
+        >>>
```
Comment (seisman) [inline source/obspy/install.rst:32]: What about running `python -c "import obspy; print(obspy.__version__)"`?

**Review by seisman — APPROVED** [review]

**Comment by seisman** [comment]
@core-man When you merge a PR, you will see a long list of commits and some "Co-authored-by" lines. You can clean up the long commits history, but please leave the "Co-authored-by" lines to give the reviewers (who made suggestions) more credits.

See the difference between the latest two commits:

image

**Comment by core-man** [comment]
ggg~ If Co-authored-by several times, shall I only keep one?

**Comment by seisman** [comment]
> ggg~ If Co-authored-by several times, shall I only keep one?

Yes, keeping one line for each reviewer.

**Comment by core-man** [comment]
> > ggg~ If Co-authored-by several times, shall I only keep one?
> 
> Yes, keeping one line for each reviewer.

Okay~  Sorry for that~

**Comment by seisman** [comment]
I don't care about that, but just in case some future reviewers want more credits.

**Pull request merged**